include/argparse/argparse.hpp

```diff
@@ -1018,7 +1018,7 @@ class ArgumentParser {
       stream << "Positional arguments:\n";
 
     for (const auto &mPositionalArgument : parser.mPositionalArguments) {
-      stream.width(tLongestArgumentLength);
+      stream.width(static_cast<std::streamsize>(tLongestArgumentLength));
```

Comment:
Yes, it will be no longer needed. I think would be better to just change variable `tLongestArgumentLength` on line [1007](https://github.com/p-ranav/argparse/blob/master/include/argparse/argparse.hpp#L1007) to std::streamsize

Reply:
like so
```C++
const auto& tLongestArgumentLength = static_cast<std::streamsize>(parser.get_length_of_longest_argument());
```

Reply:
Why do you think it is better to `static_cast` instead of using the expected final type from the beginning? I'm not understanding the advantage you see.